techtv2ovs/management/commands/techtvimport.py

```diff
@@ -0,0 +1,72 @@
+""" techtv2ovs command """
+from bonobo.contrib.django import ETLCommand
+from django.conf import settings
+
+from techtv2ovs.utils import TechTVImporter
+
+
+class Command(ETLCommand):
+    """
+    Django command for importing TechTV collections and videos via Bonobo ETL Graph
+    """
+
+    def add_arguments(self, parser):
+        """
+        Arguments for MySQL connection parameters
+        """
+        parser.add_argument(
+            '--host',
+            dest='host',
+            help='MySQL host',
+            type=str
+        )
+        parser.add_argument(
+            '--db',
+            dest='db',
+            help='MySQL database',
+            type=str
+        )
+        parser.add_argument(
+            '--user',
+            dest='user',
+            help='MySQL user',
+            type=str
+        )
+
+        parser.add_argument(
+            '--protected',
+            dest='protected',
+            help='filter by protected status (value 0 or 1)',
```

Comment:
Can you add a check somewhere to verify that the value is `0` or `1`?

Reply:
OK